src/configurations/dev.config.json

```diff
@@ -3,15 +3,16 @@
         "downloadUrl": "https://github.com/Microsoft/sqltoolsservice/releases/download/v{#version#}/microsoft.sqltools.servicelayer-{#fileName#}",
         "version": "0.1.4",
         "downloadFileNames": {
-            "Windows": "win-x64-netcoreapp1.0.zip",
-            "OSX": "osx-x64-netcoreapp1.0.tar.gz",
-            "CentOS": "centos-x64-netcoreapp1.0.tar.gz",
-            "Debian": "debian-x64-netcoreapp1.0.tar.gz",
-            "Fedora": "fedora-x64-netcoreapp1.0.tar.gz",
-            "OpenSUSE": "opensuse-x64-netcoreapp1.0.tar.gz",
-            "RHEL": "rhel-x64-netcoreapp1.0.tar.gz",
-            "Ubuntu14": "ubuntu14-x64-netcoreapp1.0.tar.gz",
-            "Ubuntu16": "ubuntu16-x64-netcoreapp1.0.tar.gz"
+            "Windows_7_64": "win-x64-netcoreapp1.0.zip",
+            "Windows_7_86": "win-x86-netcoreapp1.0.zip",
```

Comment:
We may not want to support this. C# doesn't and we're moving away from this in general. Given the very small number of users, I think we should take this out.